dynamic/Dockerfile

```diff
@@ -73,7 +73,7 @@ LABEL name="instana-agent" \
 ARG TARGETPLATFORM='linux/amd64'
 
 RUN microdnf upgrade \
-    && microdnf install --nodocs --noplugins hostname procps iproute util-linux python3 jq findutils less tar gzip socat \
+    && microdnf install --nodocs --noplugins podman crun hostname procps iproute util-linux python3 jq findutils less tar gzip socat \
```

Comment:
Is podman available on all environments?
Is there any cleanup required after podman/crun install like we do for docker?

Reply:
Yes, as per the Podman installation documentation, it's available for all arches. We just need to change the package manager based on the platform. Here is the doc: https://podman.io/getting-started/installation.We are using the package manager microdnf which is available for env and about cleanup, we don't need to perform any cleanup like other containers clients(docker, containerd, etc) because other containers the client contains docs with releases. In the case of Podman, we have just Podman pkg nothing else.